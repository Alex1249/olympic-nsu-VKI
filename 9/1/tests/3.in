the goal of the game is to rotate the faces of the cube in such a 


way that at the end each face has a different color as shown in figure 
notice that when a face is rotated the configuration of colors in all 

the adjacent faces changes figure illustrates a rotation of one of the 
  ces given a scrambled configuration reaching the final position can 
be quite challenging as you may know

