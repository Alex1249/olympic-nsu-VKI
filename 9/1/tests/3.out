1 - 5
2 - 15
3 - 18
4 - 11
5 - 7
6 - 5
7 - 2
8 - 4
9 - 2
11 - 2
13 - 2
